config/phpstan/static-reflection.neon

```diff
@@ -3,15 +3,10 @@ parameters:
     featureToggles:
         disableRuntimeReflectionProvider: false
 
-conditionalTags:
-    PhpParser\NodeVisitor\ParentConnectingVisitor:
-        phpstan.parser.richParserNodeVisitor: true
```

Comment:
no more PHPStan `conditionalTags` to fill parent node early 🎉 🎉 🎉

Reply:
@staabm it reduce unnecessary set parent attribute initially, avoid double parent set